torchtune/modules/rms_norm.py

```diff
@@ -37,8 +35,9 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             torch.Tensor: The normalized and scaled tensor having the same shape as ``x``.
         """
         # computation is in fp32
-        x_fp32 = x.float()
-        x_normed = (
-            x_fp32 * torch.rsqrt(x_fp32.pow(2).mean(-1, keepdim=True) + self.eps)
-        ).type_as(x)
-        return x_normed * self.scale
+        return F.rms_norm(
+            x.float(),
+            normalized_shape=self.normalized_shape,
+            weight=self.scale,
+            eps=self.eps,
```

Comment:
noob question: when we make the model bf16, will self.eps and self.scale also become bf16 or do they stay float32?

If they are bf16, does it influence the rms_norm?

I was checking and the torch.nn version asks for dtype to cast scale/eps: https://pytorch.org/docs/stable/generated/torch.nn.RMSNorm.html